modules/persistent-storage-csi-about.adoc

```diff
@@ -0,0 +1,6 @@
+:_mod-docs-content-type: CONCEPT
+[id="csi-about_{context}"]
+= About CSI
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plugins using a standard interface without ever having to change the core Kubernetes code.
+
+CSI Operators give {product-title} users storage options, such as volume snapshots, that are not possible with in-tree volume plugins.
```

Comment:
```suggestion
CSI Operators give {product-title} users storage options, such as volume snapshots that are not possible with in-tree volume plugins.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases